cmake/TaichiTests.cmake

```diff
@@ -23,6 +23,14 @@ include_directories(
 )
 
 add_executable(${TESTS_NAME} ${TAICHI_TESTS_SOURCE})
+if (WIN32)
+    # Output the executable to bin/ instead of build/Debug/...
+    set_target_properties(${TESTS_NAME} PROPERTIES RUNTIME_OUTPUT_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/bin")
```

Comment:
nit: Shall we create a variable for this?

Reply:
A variable for the output place or what?

Reply:
right, for the output place